packages/remix-server-runtime/responses.ts

```diff
@@ -1,7 +1,26 @@
-export type JsonFunction = <Data>(
+type SerializablePrimitives =
+  | string
+  | number
+  | boolean
+  | null
+  | { toJSON(): any }
```

Comment:
This should be `unknown` I think?
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any

```suggestion
  | { toJSON(): unknown }
```